cms-publish-rest/src/test/java/se/simonsoft/cms/publish/rest/PublishResourceTest.java

```diff
@@ -148,9 +168,15 @@ public void publishResourceTest() throws Exception {
 		PublishProfilingSet ppSet = new PublishProfilingSet();
 		ppSet.add(recipe);
 		when(publishConfigurationMock.getItemProfilingSet(any(CmsItemPublish.class))).thenReturn(ppSet);
-		
-		PublishResource resource = new PublishResource("localhost",
+
+		AwsCredentialsProvider credentials = DefaultCredentialsProvider.create();
+		Region region = DefaultAwsRegionProviderChain.builder().build().getRegion();
+		CmsExportPrefix exportPrefix = new CmsExportPrefix(storage.getPathversion());
+		when(exportProviderMock.getReader()).thenReturn(new CmsExportProviderAwsSingle(exportPrefix, cloudId, bucketName, region, credentials).getReader());
+
+		PublishResource resource = new PublishResource("demo.simonsoftcms.se",
 														executionStatusMock,
+														exportProviderMock,
```

Comment:
Since the exportProvider is mocked it does not need to communicate with AWS.

Reply:
Yep, removed.